crates/nu-std/std/core/mod.nu

```diff
@@ -6,8 +6,8 @@ let dt = (datetime-diff (date now) 2019-05-10T09:59:12-07:00)
 
 let banner_msg = $"(ansi green)     __  ,(ansi reset)
 (ansi green) .--\(\)°'.' (ansi reset)Welcome to (ansi green)Nushell(ansi reset),
-(ansi green)'|, . ,'   (ansi reset)based on the (ansi green)nu(ansi reset) language,
-(ansi green) !_-\(_\\    (ansi reset)where all data is structured!
+(ansi green)'|, . ,'   (ansi reset)where all data is structured!
+(ansi green) !_-\(_\\    (ansi reset)v((version).version) ((version).build_os) 
```

Comment:
it's probably not a good idea to run version twice. we could just run it, store it in a variable and reuse it, right?

Reply:
Good point - Fixed in latest commit.